Integrations/integration-QRadar.yml

```diff
@@ -448,4 +453,4 @@ script:
       description: Number of results in return
     description: Get closing reasons
   isfetch: true
-releaseNotes: 'Adding proxy config support for Qradar'
\ No newline at end of file
+releaseNotes: 'Printing message body in case of stringify error'
```

Comment:
don't you need to add to releaseNotes that was there? instead of override?

Reply:
No, and tests failed because of macafee, already opened issue